hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2021,6 +2021,14 @@
       This fallback approach is not recommended for production environments.
     </description>
   </property>
+  <property>
+    <name>ozone.om.ratis.snapshot.max.total.sst.size</name>
+    <value>100000000</value>
```

Comment:
What the unit  (MB/GB) ?

Reply:
It is just bytes.   Should be MB.  I'll change it.